force-app/main/default/classes/SOQL.cls

```diff
@@ -96,9 +111,10 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByEmailDomain method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> getContactsByEmailDomain(){
+     List<Contact> contactsByEmailDomain = [SELECT Id, Name, Email FROM Contact WHERE Email LIKE '%example.com%'];
```

Comment:
Careful with this one, you do not want to return someone with the email example.com@gmail.com